website/.storybook/main.js

```diff
@@ -1,35 +1,23 @@
 const path = require("path");
 
 module.exports = {
-  stories: [
-    "../src/components/**/*.stories.mdx",
-    "../src/components/**/*.stories.@(js|jsx|ts|tsx)",
-  ],
+  stories: ["../src/components/**/*.stories.mdx", "../src/components/**/*.stories.@(js|jsx|ts|tsx)"],
   addons: [
     "@storybook/addon-links",
     "@storybook/addon-essentials",
     "@storybook/addon-interactions",
     "@chakra-ui/storybook-addon",
-    "storybook-addon-next-router",
   ],
-  framework: "@storybook/react",
-  core: {
-    builder: "@storybook/builder-webpack5",
-  },
+  framework: "@storybook/nextjs",
   staticDirs: ["../public"],
   // https://github.com/storybookjs/storybook/issues/15336#issuecomment-888528747
   typescript: { reactDocgen: false },
-  // fix to make absolute imports working in storybook
-  webpackFinal: async (config, { configType }) => {
-    config.resolve.alias = {
-      ...config.resolve.alias,
-      src: path.resolve(__dirname, "../src"),
-      styles: path.resolve(__dirname, "../styles"),
-    };
```

Comment:
This is not needed anymore as [storybook now works with next.js automatically](https://storybook.js.org/blog/integrate-nextjs-and-storybook-automatically/).